packages/bz2.rb

```diff
@@ -2,61 +2,33 @@
 
 class Bz2 < Package
   description 'bzip2 is a freely available, patent free, high-quality data compressor.'
-  homepage 'http://www.bzip.org/'
-  version '1.0.8'
+  homepage 'https://gitlab.com/federicomenaquintero/bzip2/'
+  version '1.0.7-6211'
```

Comment:
Downgrade?  What is the purpose?

Reply:
Switching upstream to a more modern repo.

```
bzip2/libbzip2 version 1.0.8 of 13 July 2019
Copyright (C) 1996-2019 Julian Seward <jseward@acm.org>
```

Vs

```
Copyright (C) 1996-2010 Julian Seward jseward@acm.org
Copyright (C) 2019 Federico Mena Quintero federico@gnome.org
```

Also

```
Maintainers
Since June 2019, the maintainer of Bzip2/libbz2 is Federico Mena
Quintero.  Feel free to contact me for any questions you
may have about Bzip2, both its usage and its development.  You can
contact me in the following ways:


Mail me at federico@gnome.org.


IRC: I am federico on irc.gnome.org in the #rust or
#gnome-hackers channels.  I'm there most weekdays (Mon-Fri)
starting at about UTC 14:00 (that's 08:00 my time; I am in the UTC-6
timezone).  If this is not a convenient time for you, feel free to
mail me and we can arrange a time.



Special thanks
Federico would like to thank Julian Seward, the original author of
Bzip2/libbz2, for creating the program and making it a very compelling
alternative to previous compression programs back in the early
2000's.  Thanks to Julian also for letting Federico carry on with the
maintainership of the program.
```

Maybe I should get on IRC and ask why he hasn't revved the version to 1.0.8?

Reply:
Yes, that doesn't make sense he would be working with the 1.0.7 codebase.

Reply:
Related: https://gitlab.com/federicomenaquintero/bzip2/-/issues/30

Reply:
I looked at the GitLab repo and it appears the only additional code was related to tests.  It probably makes sense to use https://sourceware.org/bzip2/ until the GitLab repo adds relevant new code and starts tagging releases.

Reply:
I agree.